### PR TITLE
Add JupyterHub Connector to Dataverse guide

### LINKED ISSUE #10021: Dataverse guide addition : JupyterHub connector External Tool
**Overview of the Feature Request**
We developped a React based connector to transfer data between a Dataverse instance and a JupyterHub serveur using Dataverse's External Tools feature, the Dataverse Rest API and JupyterHub APIs.
Here is the [repository](https://forgemia.inra.fr/dipso/eosc-pillar/dataverse-jupyterhub-connector).

We are proposing the folowing description to add to the Integrations section of the guides, in the Analysis and Computation category:

*JupyterHub*
The [Dataverse-to-JupyterHub Data Transfer Connector](https://forgemia.inra.fr/dipso/eosc-pillar/dataverse-jupyterhub-connector) streamlines data transfer between Dataverse repositories and the cloud-based platform JupyterHub, enhancing collaborative research. This connector facilitates seamless two-way transfer of datasets and files, emphasizing the potential of an integrated research environment. It is a lightweight client-side web application built using React and relying on the Dataverse External Tool feature, allowing for easy deployment on modern integration systems. Currently, it supports small to medium-sized files, with plans to enable support for large files and signed Dataverse endpoints in the future. 

**What kind of user is the feature intended for?**
The feature is intended for reasearchers, scientists and data analyst working with Dataverse instances and JupyterHub looking to ease the data transfer process.

### PR DESCRIPTION
**Pull request by sbondka**
- closes #10021

preview at https://dataverse-guide--10153.org.readthedocs.build/en/10153/admin/integrations.html#jupyterhub

### CODE REVIEW

File: doc/sphinx-guides/source/admin/integrations.rst [hunk 1]
```diff
@@ -185,6 +185,16 @@ Avgidea Data Search
...
+JupyterHub
+++++++++++
+
+The Dataverse-to-JupyterHub Data Transfer Connector streamlines data transfer between Dataverse repositories and the cloud-based platform JupyterHub, enhancing collaborative research.
+This connector facilitates seamless two-way transfer of datasets and files, emphasizing the potential of an integrated research environment.
+It is a lightweight client-side web application built using React and relying on the Dataverse External Tool feature, allowing for easy deployment on modern integration systems. Currently, it supports small to medium-sized files, with plans to enable support for large files and signed Dataverse endpoints in the future.
+
+What kind of user is the feature intended for?
+The feature is intended for reasearchers, scientists and data analyst working with Dataverse instances and JupyterHub looking to ease the data transfer process.
+
```
Comment (pdurbin) [inline doc/sphinx-guides/source/admin/integrations.rst:197]: Do we also want to link to the presentation by @r1beguin? This: https://harvard.zoom.us/rec/share/0RpoN_a7HPXF9jpBovtvxVgcaEbqrv5ZBSIKISVemdZjswGxOzbalQYpjebCbLA1.y2ZjRXYxhq8C_SU7

File: doc/sphinx-guides/source/admin/integrations.rst [hunk 2]
```diff
@@ -185,6 +185,16 @@ Avgidea Data Search
...
 
+JupyterHub
+++++++++++
+
+The Dataverse-to-JupyterHub Data Transfer Connector streamlines data transfer between Dataverse repositories and the cloud-based platform JupyterHub, enhancing collaborative research.
+This connector facilitates seamless two-way transfer of datasets and files, emphasizing the potential of an integrated research environment.
+It is a lightweight client-side web application built using React and relying on the Dataverse External Tool feature, allowing for easy deployment on modern integration systems. Currently, it supports small to medium-sized files, with plans to enable support for large files and signed Dataverse endpoints in the future.
+
+What kind of user is the feature intended for?
+The feature is intended for reasearchers, scientists and data analyst working with Dataverse instances and JupyterHub looking to ease the data transfer process.
```
Comment (pdurbin) [inline doc/sphinx-guides/source/admin/integrations.rst:196]:
```suggestion
The feature is intended for researchers, scientists and data analyst working with Dataverse instances and JupyterHub looking to ease the data transfer process.
```

File: doc/sphinx-guides/source/admin/integrations.rst [hunk 3]
```diff
@@ -185,6 +185,16 @@ Avgidea Data Search
 
 Researchers can use a Google Sheets add-on to search for Dataverse installation's CSV data and then import that data into a sheet. See `Avgidea Data Search <https://www.avgidea.io/avgidea-data-platform.html>`_ for details.
 
+JupyterHub
+++++++++++
+
+The Dataverse-to-JupyterHub Data Transfer Connector streamlines data transfer between Dataverse repositories and the cloud-based platform JupyterHub, enhancing collaborative research.
```
Comment (pdurbin) [inline doc/sphinx-guides/source/admin/integrations.rst:191]: We should add a link, right? To https://forgemia.inra.fr/dipso/eosc-pillar/dataverse-jupyterhub-connector ?

File: doc/sphinx-guides/source/admin/integrations.rst [hunk 4]
```diff
@@ -185,6 +185,16 @@ Avgidea Data Search
...
+This connector facilitates seamless two-way transfer of datasets and files, emphasizing the potential of an integrated research environment.
+It is a lightweight client-side web application built using React and relying on the Dataverse External Tool feature, allowing for easy deployment on modern integration systems. Currently, it supports small to medium-sized files, with plans to enable support for large files and signed Dataverse endpoints in the future.
```
Comment (pdurbin) [inline doc/sphinx-guides/source/admin/integrations.rst:193]: I found the external tool manifest: https://forgemia.inra.fr/dipso/eosc-pillar/dataverse-jupyterhub-connector/-/blob/master/externalTools.json

We should also edit https://dataverse-guide--10153.org.readthedocs.build/en/10153/admin/external-tools.html#inventory-of-external-tools to add the tool there.

**Review by pdurbin — COMMENTED** [review]
@sbondka thanks for the pull request! I'm leaving some comments and questions.

File: doc/sphinx-guides/source/admin/integrations.rst (hunk 1 above)
Comment (sbondka) [reply doc/sphinx-guides/source/admin/integrations.rst:197]: A link for the presentation was added.

File: doc/sphinx-guides/source/admin/integrations.rst (hunk 2 above)
Comment (sbondka) [reply doc/sphinx-guides/source/admin/integrations.rst]: Correction done.

File: doc/sphinx-guides/source/admin/integrations.rst (hunk 3 above)
Comment (sbondka) [reply doc/sphinx-guides/source/admin/integrations.rst]: The link "https://forgemia.inra.fr/dipso/eosc-pillar/dataverse-jupyterhub-connector" was added.

File: doc/sphinx-guides/source/admin/integrations.rst (hunk 4 above)
Comment (sbondka) [reply doc/sphinx-guides/source/admin/integrations.rst:193]: Description was added to "https://dataverse-guide--10153.org.readthedocs.build/en/10153/admin/external-tools.html#inventory-of-external-tools"

**Comment by r1beguin** [comment]
I developped this JupyterHub connector. Thanks for the corrections and additions on this doc, everything looks good to me !

**Pull request merged**